src/patternfly/components/Table/table-scrollable.scss

```diff
@@ -13,12 +12,21 @@
 
   .pf-c-table__sticky-column {
```

Comment:
unrelated - does this need to be scoped in `.pf-c-table?`

Reply:
Yes, it needs a higher specificity